src/assets/locales/fr.json

```diff
@@ -77,6 +103,7 @@
   "chat-sidebar.log-message.unauthorized-scene-change": "Vous n'avez pas l'autorisation de changer de scène.",
   "chat-sidebar.system-message.entered-lobby": "{name} est entré(e) dans le hall d'attente.",
   "chat-sidebar.system-message.entered-room": "{name} est entré(e) dans la salle.",
+  "chat-sidebar.system-message.hub-change": "Vous êtes maintenant dans {hubName}",
```

Comment:
"chat-sidebar.system-message.hub-change": "You are now in {hubName}",

Reply:
Suggestion: "Vous êtes à présent dans"